matching/time_matcher.go

```diff
@@ -48,6 +53,28 @@ type PlaceCluster struct {
 	Slot   TimeSlot `json:"time slot"`
 }
 
+func (thisquerytime *QueryTimeStart) AddOffsetHours(offsethour uint8) (newquerytime QueryTimeStart, valid bool){
+	newquerytime.Day = thisquerytime.Day
+	newquerytime.StartHour = thisquerytime.StartHour
```

Comment:
these assignment is excessive, first check if adding the offset is valid then assign.